readme.md

```diff
@@ -271,7 +271,12 @@ const log = config => (set, get, api) => config(args => {
 }, get, api)
 
 // Turn the set method into an immer proxy
-const immer = config => (set, get, api) => config(fn => set(produce(fn)), get, api)
+const immer = config => (set, get, api) => config((partial, replace) => {
+  const nextState = typeof partial === 'function'
+      ? produce(partial)
+      : produce(() => partial)
```

Comment:
How is it different from this?
```suggestion
      partial
```

Reply:
Yea you right. There is no difference. I updated the code accordingly.